chaste_codegen/chaste_model.py

```diff
@@ -11,15 +11,14 @@
     Expr,
     Float,
     Function,
-    Mul,
     Pow,
     Wild,
     log,
     simplify,
     sympify,
 )
-from sympy.codegen.cfunctions import log10
-from sympy.codegen.rewriting import ReplaceOptim, optimize, optims_c99
+from sympy.codegen.cfunctions import log2, log10
+from sympy.codegen.rewriting import ReplaceOptim, optimize
```

Comment:
What does `optims_c99` do that we're now missing out?

Reply:
optims_c99 = (expm1_opt, log1p_opt, exp2_opt, log2_opt, log2const_opt)

Reply:
Worth getting some data for each of these as to how often they can be applied in our full suite of models, and how expensive they are to apply, to guide which ones we decide are worth doing.

Reply:
optims_c99 = (expm1_opt, log1p_opt, exp2_opt, log2_opt, log2const_opt)

expm1_opt -  exp(x-1) --> expm1(x) -- might want it back I'll check if it influences the output
log1p_opt - log(x+1) --> log1p(x) -- might want it back I'll check if it influences the output
exp2_opt - Pow(x, 2) --> exp2(x) -- might want it back I'll check if it influences the output
log2_opt - log(x)/log(2) -> log2(x) -- doesn't make a difference to the output
log2const_opt - log(2)*log2(_w) -> log(_w) -- doesn't make a difference to the output